### PR TITLE
Internal: Editor V2 - Run on Elementor root build

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Review by StyleShit — DISMISSED** [review]

File: packages/.eslintrc.js [hunk 1]
```diff
@@ -26,6 +28,12 @@ module.exports = {
 				message: 'Path import of Elementor dependencies is not allowed, please use the package root (e.g: use "@elementor/locations" instead of "@elementor/locations/src/index.ts").',
 			},
 		],
+		'import/no-extraneous-dependencies': [
+			'error',
+			{
+				devDependencies: false,
```
Comment (StyleShit) [inline packages/.eslintrc.js:34]: explain

File: packages/.eslintrc.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ module.exports = {
 		'plugin:import/typescript',
 	],
 	settings: {
+		// Override import/internal-regex that defined in @wordpress/eslint-plugin.
```
Comment (StyleShit) [inline packages/.eslintrc.js:13]:
```suggestion
		// Override `import/internal-regex` that is defined in `@wordpress/eslint-plugin`.
```

File: packages/.eslintrc.js [hunk 3]
```diff
@@ -42,7 +50,9 @@ module.exports = {
 		{
 			// Development files.
 			files: [
-				'**/@(__mocks__|__tests__|tests|test)/**/*.[tj]s?(x)',
+				'**/tools/**/*', // Tools files.
+				'*.js', // Root level files.
```
Comment (StyleShit) [inline packages/.eslintrc.js:54]: ts?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: packages/.eslintrc.js (hunk 1 above)
Comment (Nevoss) [reply packages/.eslintrc.js:34]: Done

File: packages/.eslintrc.js (hunk 2 above)
Comment (Nevoss) [reply packages/.eslintrc.js:13]: Done

File: packages/.eslintrc.js (hunk 3 above)
Comment (Nevoss) [reply packages/.eslintrc.js:54]: Done

**Review by StyleShit — APPROVED** [review]

**Review by rotemee — APPROVED** [review]

**Pull request merged**